app/datatables/protocols_datatable.rb

```diff
@@ -144,6 +144,7 @@ def get_raw_records_base
                                      .select(:id)
     published_versions = @team.protocols
                               .where(protocol_type: Protocol.protocol_types[:in_repository_published_version])
+                              .order(:parent_id, version_number: :desc)
```

Comment:
why do you need order here?

Reply:
To show the latest published version on the screen. In other cases, we can show incorrect data connected to selected records (eg. keywords,...). As I understand, DISTINCT ON selects the first row, so if we do not order before, we can not guarantee that we choose the row of the last published protocol.

Reply:
Right, missed that